pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/store/IndexKey.java

```diff
@@ -26,11 +26,11 @@
 /**
  * Class representing index name and type
  */
-public class IndexKey {
+public class IndexKey implements Comparable<IndexKey> {
   private static Logger LOGGER = LoggerFactory.getLogger(IndexKey.class);
 
-  String name;
-  ColumnIndexType type;
+  final String name;
```

Comment:
Fix the code style

Reply:
reformated those in another quick PR: https://github.com/apache/pinot/pull/7316/files